modules/matchPattern.js

```diff
@@ -1,22 +1,20 @@
 import pathToRegexp from 'path-to-regexp'
 
-const cache = {}
+// cache[exactly][pattern] contains getMatcher(pattern, exactly)
+const cache = {true: {}, false: {}}
```

Comment:
So, this is the only weird part that bothers me. But implicit type conversion always does that for me...